pkg/sql/schema_changer.go

```diff
@@ -1758,29 +1757,27 @@ func (sc *SchemaChanger) done(ctx context.Context) error {
 
 		// Clean up any comments related to the mutations, specifically if we need
 		// to drop them.
-		metaDataUpdater := descmetadata.NewMetadataUpdater(ctx,
-			sc.ieFactory,
-			descsCol,
-			&sc.settings.SV,
-			txn,
-			NewFakeSessionData(&sc.settings.SV))
 		for _, comment := range commentsToDelete {
-			err := metaDataUpdater.DeleteDescriptorComment(
-				comment.id,
-				comment.subID,
-				comment.commentType)
-			if err != nil {
+			if err := descsCol.DeleteComment(
+				ctx, false /* kvTrace */, b, descpb.ID(comment.id), uint32(comment.subID), comment.commentType,
```

Comment:
Should `kvTrace` be set to `p.ExtendedEvalContext().Tracing.KVTracingEnabled()`?

Reply:
No problem at all.